packages/core/core-flows/src/cart/workflows/update-cart.ts

```diff
@@ -68,7 +68,11 @@ export const updateCartWorkflow = createWorkflow(
     const cartInput = transform(
       { input, region, customerData, salesChannel, cartToUpdate },
       (data) => {
-        const { promo_codes, ...updateCartData } = data.input
+        const {
+          promo_codes,
+          additional_data: _,
```

Comment:
Q: any reason we create a new var `_` to store additional data here? wouldn't just restructure be enough here?

Reply:
+!

Reply:
That just removes additional_data property from updateCartData variable.
Elegant solution 😎

Reply:
If you don't put `_` it will complain that the unstructured property is unused. We have disabled `noUnusedParameters` in our `tsconfig.json` so it might not be obvious right away.

https://www.typescriptlang.org/play/?allowUnusedLabels=true&noUnusedLocals=true&noUnusedParameters=true#code/KYDwDg9gTgLgBAYwgOwM7wJbLAV3gXjgG8AoORAQ1gC5iBfAGjLgoBNWMYMUKAbAfVYUYFWkTokJJJGnhFKsBi3aduyPoOEU4dOISy4Y0qjCA

Reply:
I dont think it is true when you have a spread operator after the destructured properties

Reply:
Ahuh, interesting. I didn't know that :)